samples/mongobetween_sample/README.md

```diff
@@ -0,0 +1,238 @@
+# Steps to run a MongoBetween connection proxy with Amazon DocumentDB
+
+## Introduction
+
+### Why do we need a connection proxy for Amazon DocumentDB
+
+The maximum number of connections that DocumentDB currently supports is 30K per instance, and this is a hard limit. Scenarios where we see customers hitting these limits are with applications that are deployed using containers and have connections that are opened for a prolonged period of time. A common design solution to minimize the connections created on the database instances is to use a connection proxy layer. In this article, we introduce an open-source proxy solution and provide steps to install and deploy this proxy, along with the required application changes.
+
+#### What is Mongobetween?
+
+Mongobetween is a lightweight MongoDB connection pooler written in Golang. Its primary function is to handle a large number of incoming connections and multiplex them across a smaller connection pool to one or more DocumentDB clusters.
```

Comment:
I don't understand the second sentence, how/why would you use it for >1 DocumentDB clusters?

Reply:
mongobetween can also act as a proxy between multiple documentdb clusters. this use case is not covered here